### DISCUSSION TITLE
Starting Today

### DISCUSSION THREAD

**Discussion by jangirvipin** [Cyfrin/foundry-full-course-cu#2423] [General]
I am starting today my journey of web3 . Can someone tell me the prerequisite of web3 ?

**Comment by EngrPips** [comment]
Hello @jangirvipin, Welcome aboard, and we wish you the best on your journey. Start with the blockchain basics course on cyfrin updraft and you are on the right track.

**Reply by Depdacker** [reply]
I'm starting from scratch, can you please explain what's the use of blockchains?

**Reply by EngrPips** [reply]
Patrick explained everything here  https://updraft.cyfrin.io/ . Start with `blockchain basics`

**Reply by jangirvipin** [reply]
i am also starting from scratch . I have basic knowledge of web2 and now i am deep diving in web3. 
Blockchain is used to make decentralized payments ,it means that no one have complete control on our transaction and data unlike the real world payments we made through banks.

**Comment by Anshpatwal** [comment]
> I am starting today my journey of web3 . Can someone tell me the prerequisite of web3 ?

Hey Congrats on starting today , I also started 2 days ago and the course by cyfrin updraft is one bof the best you can get , well good luck!

**Comment by EngrPips** [comment]
Good luck, everyone.

**Comment by PhilippeOurselin** [comment]
Welcome abroad! This is my third week here ;) In addition to this excellent course, I advise you to follow Alchemy University Javascript course, since Javascript is also a must-have programming language in Web3 universe: in fact, Solidity is mainly inspired from Javascript. By enhancing your Javascript, you will also enhance your Solidity.

https://university.alchemy.com/overview/js

**Comment by EngrPips** [comment]
Interesting.

**Comment by ricardomazuera** [comment]
@jangirvipin Good luck on your journey. Prerequisites Patrick teaches you in [this](https://updraft.cyfrin.io/courses/blockchain-basics) course.
